openssh/README.md

```diff
@@ -0,0 +1,106 @@
+## Purpose 
+
+This directory contains a Dockerfile that builds the [OQS OpenSSH fork](https://github.com/open-quantum-safe/openssh), which allows to establish a quantum-safe SSH connection using quantum-safe keys and quantum-safe authentication.
+
+## Quick start
+
+[Install Docker](https://docs.docker.com/install) and run the following commands in this directory:
+
+1. Run `docker build -t oqs-openssh-img .` This will generate the image with the name `oqs-openssh-img`
+2. `docker run --name oqs-openssh-server -ditp 2222:2222 --rm oqs-openssh-img`
+This will start a docker container that has sshd listening for SSH connections on port 2222 and this port is forwarded to and accessible via `localhost:2222`.
+3. `docker run --rm --name oqs-openssh-client -dit oqs-openssh-img` will start a docker container with the same properties as `oqs-openssh-server` except the port 2222 is not published.
+4. You can hop on either of those two containers as a non-root user (oqs) to use the built-in OQS-OpenSSH binaries or do other shenanigans by typing
+`docker exec -ti -u oqs oqs-openssh-server /bin/sh`
+Of course adjust the container's name accordingly if hopping onto the client (==> `oqs-openssh-client`).
```

Comment:
again, suggest using a "cut-copy-paste" ready line instead of just 'oqs-openssh-client'